packages/apptive_grid_core/lib/src/model/form/form_data.dart

```diff
@@ -210,5 +215,106 @@ class FormDataProperties {
         successMessage,
         buttonTitle,
         reloadAfterSubmit,
+        afterSubmitAction,
+      );
+}
+
+/// The AfterSubmitAction class represents an action to be taken after a form
+/// is submitted.
+class AfterSubmitAction {
```

Comment:
The whole `AfterSubmitAction` should be lifted in a separate file and necessary objects should be exposed in the library

Reply:
done